### ISSUE TITLE
🐞  FF autosave post/page revisions error unwantingly.

### ISSUE THREAD

**Issue by tradesouthwest** [ClassicPress/ClassicPress#1549]
### Expected behavior

FF v129 & higher save post/pages without any errors.

### Current behavior

FF v129 & higher returns error message in post/page editor which reappears even after save.

### Possible solution

Error message ONLY ocurrs in FireFox browsers. Was able to "fix" by resetting `define('AUTOSAVE_INTERVAL', 15 );` But still reoccurred in some saving behaviors. Also see https://classicpress.zulipchat.com/#narrow/stream/399215-core/topic/General/near/471965448 

### Steps to reproduce bug

It originally appeared as if it were some type of version only issue. But has been tested positive on CP v1.7.3, using Firefox and other browsers. Only FF returns issue.

### ClassicPress version

2.2.0

### PHP version

8.1

### Can you help?

Yes, I will submit a PR

**Comment by KTS915** [comment]
I think the problem lies in ` ~/wp-includes/js/autosave.js`. For me, changing line 452 from
```
return string.toString().replace(/[\x20\t\r\n\f]+/g, '');
```
to
```
return string.toString().replace(/[\x20\t\r\n\f]+/g, '').replace(/\s+/g, '').replace(/&nbsp;+/g, '');
```
resolves the problem.

**Comment by tradesouthwest** [comment]
The above comment seems to work fine in 2.2.0 w/ FF
![Screenshot_2024-09-21_12-27-04](https://github.com/user-attachments/assets/749e1433-7465-495f-92b5-fca06aebea33)

**Comment by tradesouthwest** [comment]
Javascript code update DID help. But error will reappear if you do not WAIT 60 seconds BEFORE saving. (I waited for the Publish button to flicker while counting to 60.)

**Comment by KTS915** [comment]
I'm not experiencing any need to wait before saving; the code works either way.

**Comment by tradesouthwest** [comment]
It will only error, upon not waiting for autosave; if using non HTTPS protocol. I purposely disabled 'Enable HTTPS-Only Mode in all windows' in FF Settings. Enabling it achieves successful save.

**Comment by tradesouthwest** [comment]
Also validate that new code in line 452 DOES work, by commenting out new code and putting original back in place. Clear cookies and save will throw error. But when I put the new line of code in place, pages/post save without any error message. 

Will wait to mark complete pending any core update from other team members.

**Comment by xxsimoxx** [comment]
Closed by #1575 .
